spec/test_app_templates/Gemfile.extra

```diff
@@ -1 +1,3 @@
-gem 'hyrax'
```

Comment:
I can't remember why but I needed to keep the hyrax gem here to ensure it was explicitly added to the test app's Gemfile.  If it works without great!  I wouldn't be surprised if we had to add it back at some point.